packages/lesmis/src/views/SurveyResponsesView.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import {
+  SurveyResponsesPage as ResourcePage,
+  SURVEY_RESPONSE_COLUMNS,
+} from '@tupaia/admin-panel/lib';
+import { ConfirmModal } from '@tupaia/ui-components';
+import { ApproveButton } from '../components';
+
+// Todo: Replace base filter with real filter @see WAI-832
+export const ApprovedSurveyResponsesView = props => (
+  <ResourcePage
+    title="Approved Survey Responses"
+    baseFilter={{ 'survey.code': { comparator: 'ILIKE', comparisonValue: '%_Confirmed_WNR' } }}
+    {...props}
```

Comment:
Does this need columns passsed in?

Reply:
No it doesn't as `ResourcePage` here is actually the whole `SurveyResponsesPage` from admin-panel. The way I've named it `ResourcePage` is confusing though so I've changed it to simply be `SurveyResponsesPage`